torch/csrc/distributed/c10d/comm.hpp

```diff
@@ -51,6 +53,15 @@ class TORCH_API GradBucket {
   // parameter.
   std::vector<at::Tensor> getPerParameterTensors() const;
 
+  // Returns mapping of gradient index in bucket to model parameter. Practically
+  // this can be used in conjunction with getPeraParameterTensors() as follows:
+  // auto grads = getPerParameterTensors(); // grad tensors for this bucket
+  // auto mapping = getGradIndexToVariable();
```

Comment:
nit: I think perhaps `getGradIndexToVariable()` was your old version of the function name, and this should be `auto mapping = getGradIndexToVariableMapping()` now.